temporalio/client.py

```diff
@@ -831,6 +832,108 @@ async def list_schedules(
             )
         )
 
+    async def update_worker_build_id_compatability(
+        self,
+        task_queue: str,
+        operation: BuildIdOperation,
+        rpc_metadata: Mapping[str, str] = {},
+        rpc_timeout: Optional[timedelta] = None,
+    ) -> None:
+        """Used to add new Build IDs or otherwise update the relative compatibility of Build Ids as
+        defined on a specific task queue for the Worker Versioning feature.
+
+        For more on this feature, see https://docs.temporal.io/workers#worker-versioning
+
+        .. warning::
+           This API is experimental
+
+        Args:
+            task_queue: The task queue to target.
+            operation: The operation to perform.
+            rpc_metadata: Headers used on each RPC call. Keys here override
+                client-level RPC metadata keys.
+            rpc_timeout: Optional RPC deadline to set for each RPC call.
+        """
+        return await self._impl.update_worker_build_id_compatability(
```

Comment:
Eh, I kinda like this since these are supposed to always be doing the same thing, if the return type changed then this works (it wouldn't, because of compat, but I like the idea).

Reply:
No, of course not return every method. I mean it makes sense for functions which are purely proxying another function to just return that inner one, regardless of the return value - because they are purely a passthrough

Reply:
:+1: If it passes type check I guess it's just subjective here, so good enough for me